llvm/unittests/ProfileData/PGOCtxProfReaderWriterTest.cpp

```diff
@@ -115,6 +115,15 @@ TEST_F(PGOCtxProfRWTest, RoundTrip) {
     EXPECT_EQ(Ctxes.size(), 2U);
     for (auto &[G, R] : roots())
       checkSame(*R, Ctxes.find(G)->second);
+
+    DenseSet<GlobalValue::GUID> Guids;
+    Ctxes.at(1U).getContainedGuids(Guids);
```

Comment:
It is unclear to me how this test change is related to the rest of the PR.

Reply:
It's not, but this is the API that the rest of this patch hinges on. I could introduce the test separately.

Reply:
I would just commit that separately (no need for another review, that part lgtm) since that isn't a new API here.

Reply:
done (a742693f6104055ec026852a70a68275fb82f7a0)